cubano-framework/src/main/java/org/concordion/cubano/framework/ConcordionBase.java

```diff
@@ -0,0 +1,112 @@
+package org.concordion.cubano.framework;
+
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import org.concordion.api.AfterSuite;
+import org.concordion.api.BeforeExample;
+import org.concordion.api.extension.Extension;
+import org.concordion.api.option.ConcordionOptions;
+import org.concordion.api.option.MarkdownExtensions;
+import org.concordion.cubano.driver.BrowserBasedTest;
+import org.concordion.cubano.driver.web.Browser;
+import org.concordion.cubano.driver.web.provider.BrowserProvider;
+import org.concordion.ext.StoryboardExtension;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.concordion.logback.LogbackAdaptor;
+import org.junit.runner.RunWith;
+
+/**
+ * Sets up any Concordion extensions or other items that must be shared between index and test fixtures.
+ * 
+ * NOTE: Test can be run from a Fixture or an Index, any global (@...Suite) methods must be in this class
+ * to ensure the are executed from whichever class initiates the test run.
+ */
+@RunWith(ConcordionRunner.class)
+// @ConcordionResources("/customConcordion.css")
+// @Extensions({ TimestampFormatterExtension.class, RunTotalsExtension.class, ExpectedToFailInfoExtension.class })
+@ConcordionOptions(markdownExtensions = { MarkdownExtensions.HARDWRAPS, MarkdownExtensions.AUTOLINKS })
+public abstract class ConcordionBase implements BrowserBasedTest {
```

Comment:
If this is for an index page, why would it need to be a BrowserBasedTest?

Reply:
Good question! I think we need to sit down over this one and have a look at what we've done at MSD and see if any of it belongs in the framework.   This is just the base class - have another couple of classes as well that I didn't put in the project as I felt they belonged with the test project using Cubano.